src/Stratis.Bitcoin.Features.SmartContracts.Wallet/SmartContractWalletController.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using NBitcoin;
+using Stratis.Bitcoin.Connection;
+using Stratis.Bitcoin.Features.Wallet;
+using Stratis.Bitcoin.Features.Wallet.Broadcasting;
+using Stratis.Bitcoin.Features.Wallet.Controllers;
+using Stratis.Bitcoin.Features.Wallet.Interfaces;
+using Stratis.Bitcoin.Features.Wallet.Models;
+using Stratis.Bitcoin.Utilities;
+using Stratis.Bitcoin.Utilities.JsonErrors;
+
+namespace Stratis.Bitcoin.Features.SmartContracts.Wallet
+{
+    public sealed class SmartContractWalletController : WalletController
+    {
+        private readonly ILogger logger;
+
+        public SmartContractWalletController(
+            ILoggerFactory loggerFactory,
+            IWalletManager walletManager,
+            IWalletTransactionHandler walletTransactionHandler,
+            IWalletSyncManager walletSyncManager,
+            IConnectionManager connectionManager,
+            Network network, ConcurrentChain chain,
+            IBroadcasterManager broadcasterManager,
+            IDateTimeProvider dateTimeProvider) :
+            base(loggerFactory, walletManager, walletTransactionHandler, walletSyncManager, connectionManager, network, chain, broadcasterManager, dateTimeProvider)
+        {
+            this.logger = loggerFactory.CreateLogger(this.GetType().FullName);
+        }
+
+        public override IActionResult SendTransaction([FromBody] SendTransactionRequest request)
```

Comment:
No need to decorate the endpoint with 
```
[Route("send-transaction")]
[HttpPost]
```
here?

Reply:
@bokobza I have fixed this :)